app/components/common/Footer.vue

```diff
@@ -0,0 +1,484 @@
+           column: {
+             title: i18n keys on column title
+             condition: display column or not, i.e. me.isStudent()
+             lists: lists of links in footer }
+           each link: {
+             url: a.href
+             title: i18n keys on link title
+             extra: if we do not use i18n title, set this as span.spr
+             attrs: extra dom attributes setting in object, i.e. {target: '_blank'}
+           }
+
+           chinaFooter is standalone so don't worry on China resources.
+        */
+        let globalFooter = [
+          {
+            title: 'nav.general',
+            condition: true, // always display
+            lists: [
+              { url: this.cocoPath('/about'), title: 'nav.about', attrs: { 'data-event-action': 'Click: Footer About' } },
+              { url: 'https://codecombat.zendesk.com/hc/en-us', title: 'contact.faq', attrs: { target: '_blank', 'data-event-action': 'Click: Footer FAQ' } },
+              { url: this.cocoPath('/about#careers'), title: 'nav.careers' },
+              { title: 'nav.contact', attrs: { class: 'contact-modal', tabindex: -1 } },
+              { url: this.cocoPath('/parents'), title: 'nav.parent' },
+              { url: 'https://blog.codecombat.com/', title: 'nav.blog' }
+            ]
+          },
+          {
+            title: 'nav.educators',
+            condition: !me.isStudent(),
+            lists: [
+              { url: '/efficacy', title: 'efficacy.ozaria_efficacy', hide: this.isCodeCombat},
+              { url: '/impact', title: 'nav.impact', hide: this.isOzaria },
+              { url: '/teachers/resources', title: 'nav.resource_hub' },
+              { url: '/teachers/classes', title: 'nav.my_classrooms' },
+              { url: this.ozPath('/'), title: 'new_home.try_ozaria', attrs: { 'data-event-action': 'Click: Footer Try Ozaria' }, hide: this.isOzaria},
+              { url: this.cocoPath('/'), title: 'nav.return_coco', attrs: { 'data-event-action': 'Click: Footer Return to CodeCombat' }, hide: this.isCodeCombat},
+              { url: this.cocoPath('/partners'), title: 'nav.partnerships' },
+              { url: this.cocoPath('/podcast'), title: 'nav.podcast' }
+            ]
+          },
+          {
+            title: '',
+            condition: me.isStudent(),
+            lists: []
+          },
+          {
+            title: 'nav.get_involved',
+            condition: true,
+            lists: [
+              { url: 'https://github.com/codecombat/codecombat', extra: 'Github' },
+              { url: this.cocoPath('/community'), title: 'nav.community' },
+              { url: this.cocoPath('/contribute'), title: 'nav.contribute' },
+              { url: this.cocoPath('/league'), title: 'game_menu.multiplayer_tab' },
+              { url: this.forumLink, title: 'nav.forum', attrs: { target: '_blank' }, hide: me.isStudent() || !me.showForumLink() }
+            ]
+          }
+        ]
+
+        let ChinaFooter = [
```

Comment:
we don't use capital letter as first letter in variable names. nitpick.

Also, can be const!!

Did eslint not show any warning? Also, script tag code is indented which is not preferred for new files